pkg/kubectl/cmd/patch.go

```diff
@@ -57,23 +57,23 @@ type PatchOptions struct {
 
 var (
 	patchLong = templates.LongDesc(i18n.T(`
-		Update field(s) of a resource using strategic merge patch
+		Update field(s) of a resource using strategic merge patch, a JSON merge patch, or a JSON patch.
 
 		JSON and YAML formats are accepted.
 
 		Please refer to the models in https://htmlpreview.github.io/?https://github.com/kubernetes/kubernetes/blob/HEAD/docs/api-reference/v1/definitions.html to find if a field is mutable.`))
 
 	patchExample = templates.Examples(i18n.T(`
-		# Partially update a node using strategic merge patch
+		# Partially update a node using a strategic merge patch. Specify the patch as JSON.
 		kubectl patch node k8s-node-1 -p '{"spec":{"unschedulable":true}}'
 
-		# Partially update a node identified by the type and name specified in "node.json" using strategic merge patch
-		kubectl patch -f node.json -p '{"spec":{"unschedulable":true}}'
+		# Partially update a node using a strategic merge patch. Specify the patch as YAML.
+		kubectl patch node k8s-node-1 -p $'spec:\n unschedulable: true'
 
-		# Update a container's image; spec.containers[*].name is required because it's a merge key
```

Comment:
Because I can't get it to work. I tried for half a day to do a strategic merge patch on a Pod object. I asked @pwittrock, and he said to patch a Deployment instead. That leads me to believe that you can't change the containers list of a Pod object by doing a strategic merge patch.